django_rq/queues.py

```diff
@@ -24,16 +25,17 @@ def get_commit_mode():
     return RQ.get('AUTOCOMMIT', True)
 
 
-def get_queue_class(config):
+def get_queue_class(config=None, queue_class=None):
     """
     Return queue class from config or from RQ settings, otherwise return DjangoRQ
     """
```

Comment:
Could you please add a docstring to clarify the priorities? If I'm not mistaken, its:
1. From `queue_class`
2. From `config`
3. From `RQ` defined in settings.py